data/result-exporter/swe-cne-exporter/src/main/java/com/powsybl/openrao/data/swecneexporter/SweMonitoredSeriesCreator.java

```diff
@@ -164,7 +164,7 @@ private MonitoredSeries generateMonitoredSeriesFromScratch(MonitoredSeriesCreati
     }
 
     void setInOutAggregateNodes(String networkElementId, String monitoredSeriesId, MonitoredRegisteredResource registeredResource) {
-        Branch<?> branch = sweCneHelper.getNetwork().getBranch(networkElementId);
+        Branch<?> branch = cracCreationContext.getNetworkBranches().get(networkElementId);
```

Comment:
we lose the aliases by doing it like this so we'll have to make sure it all works correctly still

Reply:
all tests, including business tests, passed